jaeger-core/src/main/java/com/uber/jaeger/senders/HttpSender.java

```diff
@@ -44,7 +46,7 @@
   private static final TProtocolFactory PROTOCOL_FACTORY = new Factory();
   private static final TSerializer SERIALIZER = new TSerializer(PROTOCOL_FACTORY);
 
-  private final HttpClient httpClient = new DefaultHttpClient();
+  private final HttpClient httpClient = new DefaultHttpClient(new PoolingClientConnectionManager());
```

Comment:
That is correct, but we are using a older version of apache httpclient where `PoolingClientConnectionManager` isn't deprecated. 
Seeing that nobody is using this, I don't see any harm in moving to the latest httpclient...